dll/win32/advapi32/advapi32.spec

```diff
@@ -631,8 +631,8 @@
 @ stdcall SystemFunction041(ptr long long) # RtlDecryptMemory
 @ stdcall TraceEvent(double ptr) ntdll.EtwTraceEvent
 @ stdcall TraceEventInstance(double ptr ptr ptr) ntdll.EtwTraceEventInstance
-@ varargs TraceMessage(ptr long ptr long) ntdll.EtwTraceMessage
-@ stdcall TraceMessageVa(double long ptr long ptr) ntdll.EtwTraceMessageVa
+@ varargs TraceMessage() ntdll.EtwTraceMessage
+@ stdcall TraceMessageVa() ntdll.EtwTraceMessageVa
```

Comment:
Please keep the parameters inside for consistency?

Reply:
I too used to assume that, then I was told not to, as it is useless for forwarders: less code to maintain...